### PR TITLE
WIP: Remove DiagramTrashFolderService

### PR DESCRIPTION
**Pull request by rrrene**
## Changes

1. Use `trash` npm package for trashing files

## Issues

Closes #YourIssueNumber

PR: #1507

## How to test the changes

> Describe how others can test your changes (not required when fixing typos, linter errors and such)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,7 +122,8 @@
     "pushserve": "^1.0.2",
     "socket.io-client": "2.1.1",
     "spectrum-colorpicker": "1.8.0",
-    "sqlite3": "4.0.6"
+    "sqlite3": "4.0.6",
+    "trash": "^6.0.0"
```
Comment (alexanderkasten) [inline package.json]: Muss das hier hinzugefügt werden?
Eigentlich sollte das durch `@process-engine/solutionexplorer.repository.filesystem` automatisch installiert werden 🤔

Comment (rrrene) [reply package.json]: :+1:

**Comment by rrrene** [comment]
@alexanderkasten re-review plz

**Comment by rrrene** [comment]
Closing, due to inability to bundle dep `trash` in Aurelia. 

Related: https://github.com/process-engine/solutionexplorer.repository.filesystem/pull/11

**Pull request closed without merging**